scalajs/src/main/scala/com/example/electronapp/Main.scala

```diff
@@ -9,10 +9,8 @@ object Main extends js.JSApp {
 
   def main(): Unit = {
 
-    val app = g.require("app").asInstanceOf[App]  // Module to control application life.
-
-    // Report crashes to our server.
-    g.require("crash-reporter").start();
+    val electron = g.require("electron");
+    val app = electron.app.asInstanceOf[App] // Module to control application life.
```

Comment:
Hi Nikita!

Thanks for the PR. I am currently on holidays with no laptop so I will
review and get back to you next week. Hope that's fine.

On Tue, 24 May 2016 05:50 Nikita Gazarov, <notifications@github.com> wrote:

> In scalajs/src/main/scala/com/example/electronapp/Main.scala
> <https://github.com/bchazalet/scalajs-electron-skeleton/pull/5#discussion_r64324942>
> :
>
> > @@ -9,10 +9,8 @@ object Main extends js.JSApp {
> >
> >    def main(): Unit = {
> >
> > -    val app = g.require("app").asInstanceOf[App]  // Module to control application life.
> > -
> > -    // Report crashes to our server.
> > -    g.require("crash-reporter").start();
> > +    val electron = g.require("electron");
> > +    val app = electron.app.asInstanceOf[App] // Module to control application life.
>
> Updating for small changes in Electron's API
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/bchazalet/scalajs-electron-skeleton/pull/5/files/f4f68078f970887b757df8b133613fab08942941#r64324942>
>